scripts/enum/claim.lua

```diff
@@ -0,0 +1,12 @@
+-----------------------------------
+-- Claim type
+-----------------------------------
+xi = xi or {}
+
+---@enum xi.claimType
+xi.claimType =
+{
+    EXCLUSIVE     = 0, -- Regular exclusive claim behavior. Only one entity can attack.
```

Comment:
Only one entity can attack -> Only one entity and related group*, or something like that